multi-agent/lib/mas/actions/providers/claude_agent/validate_vertex_connection/validate_vertex_connection.py

```
@@ -0,0 +1,124 @@
+    )
+    action_type = ActionType.VALIDATION
+    input_schema = ValidateVertexConnectionInput
+    output_schema = ValidateVertexConnectionOutput
+    version = "1.0.0"
+    tags = {"claude_agent", "vertex_ai", "validation", "gcp"}
+    elements = {(ResourceCategory.NODE.value, Identifier.TYPE)}
+
+    async def execute(
+        self,
+        input_data: ValidateVertexConnectionInput,
+        context: Optional[Dict[str, Any]] = None,
```

Ruff (0.15.15)

[warning] Unused method argument: `context`

(ARG002)